extensions/sql-migration/src/api/utils.ts

```diff
@@ -120,3 +121,14 @@ export function filterMigrations(databaseMigrations: MigrationContext[], statusF
 	}
 	return filteredMigration;
 }
+
+export function selectiDropDownIndex(dropDown: DropDownComponent, index: number) {
+	if (dropDown?.values && dropDown!.values?.length >= index - 1 && index >= 0) {
```

Comment:
If these can be undefined then the params should be `dropDown?: DropDownComponent, index?: number`